ground_truth_labeling_jobs/multi_modal_parallel_sagemaker_labeling_workflows_with_step_functions/sagemaker_ground_truth_workflows.ipynb

```diff
@@ -0,0 +1,817 @@
+{
```

Comment:
<p>Please replace these bullets, with the following re-worded bullets:</p><ul><li>If you have added yourself to the work teams created by the Cloud Formation Template, use the following URL to login to the worker portal. </li><li>If you have not added yourself to the work team, use the instructions in <a href="https://docs.aws.amazon.com/sagemaker/latest/dg/sms-workforce-management-private-console.html#add-remove-workers-sm" target="_blank">Add or Remove Workers</a> to add yourself to the following work teams: <code>smgt-workflow-first-level</code>  and <code>smgt-workflow-&nbsp;second-level</code>  . </li></ul><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/2191/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='28'/>